css/index.css

```
@@ -253,3 +254,191 @@ body {
     font-size: 14px;
   }
 }
+
+
+.carousel{
+  position:relative;
+}
+
+.product {
+	max-width: 1343px;
+  height: 613px;
+	margin: auto;
+	padding: 50px 0px 0px 0px;
+	display: grid;
+	grid-template-columns: 58% 42%;
+
+}
+
+.rating {
+  display: inline-flex;
+  align-items: center;
+}
+
+.star {
+  width: 16px;
+  height: 15.4px;
+  color:#FDBF00;
+  margin-left: -13px;
+  margin-top: 8px;
+}
+
+.rating-number {  
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 17.6px;
+  margin-left: 2px;
+  margin-top: 9.5px;
+  color: #6C6B6A;
+}
+
+.product-carousel{
+  position: relative;
+  max-width: 773px;
+  margin: auto;
+  overflow: hidden;
+  width: 100%;
+  box-shadow: 5px 5px 15px rgba(0, 0, 0, 0.13);
+  border-radius: 5px; 
+}
+
+.carousel-img{
+  display: flex;
+  transition: transform 0.3s ease;
+}
+
+.carousel-item {
+  min-width: 773px;
+  height: 563px;
+  background-size: cover;
+  background-position: center;
+}
+
+.prev, 
+.next {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  background-color: white;
+  border: 1px solid black; 
+  border-radius: 50%;
+  padding: 10px;
+  cursor: pointer;
+  width: 43px;
+  height: 45px;
+  border: 2px 0px 0px 0px;
+
+}
+
+.prev {
+  left: -20px;
+  color: black;
+  border: 2px 0px 0px 0px;
+}
+
+.next {
+  right: -20px;
+  color: black;
+  border: 2px 0px 0px 0px;
+} 
+
+
+.indicators {
+  position: absolute;
+  bottom: 10px;
+  width: 100%;
+  display: flex;
+  margin-left: 268.65px;
+  margin-bottom: 151.1px;
+}
+
+.dot {
+  height: 8px;
+  width: 16px;
+  margin: 0 5px;
+  background-color: #6C6B6A;
+  border-radius: 4px;
+}
+
+.dot.active {
+  background-color: white;
+  width: 24px;
+  height: 8px;
+  border-radius: 4px;
+}
+
+
+
+.product-derecha {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	gap:25px;
+  margin-left: 39px;
+  
```

La tabulación está rara aquí
```suggestion
  display: grid;
  grid-template-columns: 1fr 1fr;
  gap:25px;
  margin-left: 39px;
```